packages/discovery-react-components/src/components/DocumentPreview/types.ts

```diff
@@ -68,4 +68,13 @@ export interface QueryResultWithOptionalMetadata extends Omit<QueryResult, 'resu
   result_metadata?: QuerytResultMetadataWithOptionalCollectionId;
 }
 
+// Facet info
```

Comment:
nit: either delete the comment, or give an explanation of what the facet info is used for or where it's from. Right now, the comment just repeats the name of the interface

Reply:
Yes, do the second part (expand on the comment)

Reply:
updated